src/css/application.css

```diff
@@ -501,32 +501,28 @@ body {
   flex: 1 0 100%;
 }
 
-.preview__pop-out-button {
+.preview__button {
   font-family: 'FontAwesome';
   width: 16px;
   height: 18px;
   position: absolute;
-  right: 10px;
-  top: 11px;
   cursor: pointer;
   opacity: 0.25;
   z-index: 1;
 }
 
+.preview__pop-out-button {
```

Comment:
So I think it makes sense for this and `preview__reset-button` to be modifiers for `preview__button`—e.g. this would be `preview__button_reset` and the other would be `preview__button_pop-out`—WDYT?